### PR TITLE
feat(frontend): node pages & footer Chinese i18n support

### PR DESCRIPTION
**Pull request by IsQiao**
This is a follow-up PR to complete the Chinese translation of Explorer in the PR: https://github.com/near/near-explorer/pull/679

### CODE REVIEW

File: frontend/src/translations/en.global.json [hunk 1]
```diff
@@ -88,6 +88,64 @@
     }
   },
   "component": {
+    "nodes": {
+      "Validators": {
+        "Validator": "Validator",
```
Comment (think-in-universe) [inline frontend/src/translations/en.global.json:93]: let's define the keys with lower case strings such as `validator`, `fee`, `warning_tip `

**Review by think-in-universe — CHANGES_REQUESTED** [review]

File: frontend/src/translations/en.global.json (hunk 1 above)
Comment (IsQiao) [reply frontend/src/translations/en.global.json]: fied

**Review by frol — APPROVED** [review]

**Comment by think-in-universe** [comment]
thanks @IsQiao  great work!

let's also complete the translation of the Receipts section in Block details page. 

image

File: frontend/src/translations/zh-hans.global.json [hunk 2]
```diff
@@ -88,6 +88,64 @@
     }
   },
   "component": {
+    "nodes": {
+      "Validators": {
+        "validator": "验证者",
+        "fee": "费用",
+        "delegators": "委托人",
```
Comment (think-in-universe) [inline frontend/src/translations/zh-hans.global.json:95]:
```suggestion
        "delegators": "委托人数",
```

File: frontend/src/translations/zh-hans.global.json [hunk 3]
```diff
@@ -88,6 +88,64 @@
...
+        "current_epoch_start": "当前时段开始在",
+        "block": "区块",
+        "complete": "完成",
+        "remaining": "剩余"
+      },
+      "NodesContentHeader": {
+        "nodes": "节点"
+      },
+      "NodeNav": {
+        "validating": "验证中",
```
Comment (think-in-universe) [inline frontend/src/translations/zh-hans.global.json:130]:
```suggestion
        "validating": "正在验证",
```

File: frontend/src/translations/zh-hans.global.json [hunk 4]
```diff
@@ -88,6 +88,64 @@
...
+        "proposed": "建议的"
```
Comment (think-in-universe) [inline frontend/src/translations/zh-hans.global.json:131]:
```suggestion
        "proposed": "已经提议"
```

File: frontend/src/translations/zh-hans.global.json [hunk 5]
```diff
@@ -88,6 +88,64 @@
...
+      },
+      "NodesCard": {
+        "nodes_validating": "验证中的节点",
```
Comment (think-in-universe) [inline frontend/src/translations/zh-hans.global.json:134]:
```suggestion
        "nodes_validating": "正在验证的节点",
```

File: frontend/src/translations/zh-hans.global.json [hunk 6]
```diff
@@ -88,6 +88,64 @@
     }
   },
   "component": {
+    "nodes": {
+      "Validators": {
+        "validator": "验证者",
```
[the remark this answers is not in the document]
Comment (think-in-universe) [reply frontend/src/translations/zh-hans.global.json:93]: 下同

**Review by think-in-universe — CHANGES_REQUESTED** [review]
Some translations need to be updated according to our glossary: https://github.com/near-x/learn-near-community/tree/master/cn/resources/glossary

**Review by shelegdmitriy — APPROVED** [review]

**Comment by think-in-universe** [comment]
thanks @IsQiao  

I fixed some translation issues, and completed the translation of the Stats page (the series name in charts).

Now we have completed all the missing Chinese translations in PR https://github.com/near/near-explorer/pull/679 , and all the pages should now have a Chinese version. 

@frol @shelegdmitriy I think we're ready to merge now. Thanks!

**Review by think-in-universe — APPROVED** [review]

**Pull request merged**